packages/api/src/webhooks/cannon.test.ts

```diff
@@ -200,7 +200,7 @@ describe("webhook cannon", () => {
         sem.release();
       };
 
-      await server.queue.emit({
+      await server.queue.publish("events.stream",{
```

Comment:
**nitnit:** Probably for the future, I think it'd be nice to have the event topics be composed with the inner `event` keys of our events. Like here `events.stream.started`. It could even be composed automatically by the `publish` function of the queues, like grabbing the `event` field from the event data and composing with the "rote namespace" provided like:
```
publish(routeNs: string, msg: { event: string }): Promise<void> {
    const route = `${routeNs}.{msg.event}`
    ...
```
and callers would go like
```
await server.queue.publish("event", { ... event: "stream.started" ... }
```
with the final routing key becoming `event.stream.started`.

This would make it really easy to implement some "on-demand" subscription to events like via websockets or SSE in the future. But it can also be iterated later, I don't think we have to implement that now.